internal/exec/util/user_group_lookup_test.go

```diff
@@ -108,7 +112,11 @@ func TestGroupLookup(t *testing.T) {
 	if err != nil {
 		t.Fatalf("temp base error: %v", err)
 	}
-	defer os.RemoveAll(td)
+	defer func() {
```

Comment:
internal/exec/util/user_group_lookup_test: Log errors 
cfb6db2384d06be893bc735172d4b829fb1c02f1

Missing lint errors in commit details